Dataphor/DAE/Strings.resx

```diff
@@ -348,4 +348,9 @@
   <data name="SchemaObjectDescription.ServerLink" xml:space="preserve">
     <value>Server link "{0}" ({1} on {2}{3})</value>
   </data>
+  <data name="ExceptionInContext" xml:space="preserve">

```

Comment:
I cannot find anywhere in the code that this string is referenced? Is this supposed to be part of this commit?